AnnoDesigner.Core/Layout/Models/LayoutFileVersionContainer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace AnnoDesigner.Core.Layout.Models
+{
+    /// <summary>
+    /// Container with just the version information.
+    /// </summary>
+    [DataContract]
+    public class LayoutFileVersionContainer
```

Comment:
I guess LayoutFile is split into these two classes so file version can be parsed before rest of the file?

In that case I'd say you are using inheritance incorrectly and I'd suggest moving all these properties into LayoutFile.
And create completely unrelated class "LayoutFileVersion" to read just the FileVersion.

Reply:
The logic needed to be split because we need to support loading of layouts in older formats (without version info).
In fact currently 3 different formats of a layout file can be loaded.
When saving a layout it is automatically converted to the newest format.

Reply:
The LayoutFileVersionContainer is only used at first part of opening a file, to get the file version. So it should be ok to make one class for loading of just FileVersion and other for loading everything.